dallinger/command_line.py

```diff
@@ -806,21 +812,22 @@ def export(app, local):
             " --app " + app_name(id),
             shell=True)
 
-        dump_path = dump_database(id)
+    try:
+        subprocess.call([
+            "dropdb",
+            app_name(id),
+        ])
+    except Exception:
+        pass
 
-        try:
-            subprocess.check_call([
-                "pg_restore",
-                "--verbose",
-                "--no-owner",
-                "--clean",
-                "-d",
-                "dallinger",
-                os.path.join("data", id, "data.dump")
-            ])
-
-        except Exception:
-            pass
+    subprocess.call([
```

Comment:
Hmm. Okay, but if this is sensitive to version differences then there's a decent chance of a non-harmless error at some point in the future. It's definitely more work, but we should consider either checking the postgresql version and complaining early if it's wrong, or check for specific warnings that we know are okay and abort on anything else.